src/Components/Checkout.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+class Checkout extends React.Component() {
+    constructor() {
+        super()
+        this.state = {
+            firstName: '',
+            lastName: '', 
+            email: '',
+            cardNum: '',
+            zipCode: ''
+        }
+    }
+
```

Comment:
include a handleChange method that updates the Checkout component state when a user types in one of the form inputs.